app/views/home/people.html.haml

```diff
@@ -1,7 +1,7 @@
 -# people cards
 
 .grid.gap-6.md:grid-cols-2
-  - User.order(username: :asc).all.each do |user|
+  - User.order(username: :asc).with_attached_avatar.each do |user|
```

Comment:
Can we move this into the #people controller action? then we can just pass @users to the view.